src/lib/molecules/Widget/index.js

```diff
@@ -91,7 +90,7 @@ function Widget({
                 </>
             ) : (
                 <>
-                    <li className="image">{img && <img src={img} alt="" />}</li>
+                    {img && <li className="image">{img && <img src={img} alt="" />}</li>}
```

Comment:
Please remove redundant second `img` check.
```suggestion
                    {img && <li className="image"><img src={img} alt="" /></li>}
```